docs/source/examples/pennylane-ibmq-backends.myst

```diff
@@ -0,0 +1,172 @@
+    
+**Note:** Using an IBM quantum computer requires a valid IBMQ account.
+See <https://quantum-computing.ibm.com/> for instructions to create an account, save credentials, and get access to online quantum computers.
+
+First, we get our devices set up depending on whether we would like to use real hardware, or a simulator.
+
+```{code-cell} ipython3
+import qiskit
+
+USE_REAL_HARDWARE = False
+
+if qiskit.IBMQ.stored_account() and USE_REAL_HARDWARE:
+    provider = qiskit.IBMQ.load_account()
+    dev = qml.device(
+        "qiskit.ibmq",
+        wires=1,
+        backend="ibmq_qasm_simulator",
+        provider=provider
+    )
+else:
+    noise_strength = 0.05
+    dev_noise_free = qml.device("default.mixed", wires=1)
+    dev = qml.transforms.insert(
+        qml.AmplitudeDamping,
+        noise_strength
+    )(dev_noise_free)
+```
+
+With `dev` set to the desired device, we can now use PennyLane's [`mitigate_with_zne`](https://pennylane.readthedocs.io/en/stable/code/api/pennylane.transforms.mitigate_with_zne.html) function, in conjuction with a noise scaling method, and inference technique from Mitiq.
+
+```{code-cell} ipython3
+from mitiq.zne.scaling import fold_global
+from mitiq.zne.inference import RichardsonFactory
+
+scale_factors = [1, 2, 3]
+noise_scale_method = fold_global
+
+device_circuit = qml.QNode(circuit, dev)
+error_mitigated_device_circuit = qml.transforms.mitigate_with_zne(
+    device_circuit,
+    scale_factors,
+    noise_scale_method,
+    RichardsonFactory.extrapolate, 
+)
+```
+
+We can now test the mitigated version of the circuit against the unmitigated to ensure it is working as expected.
+
+```{code-cell} ipython3
+unmitigated = device_circuit()
+mitigated = error_mitigated_device_circuit()
+print(f"Unmitigated result {unmitigated:.3f}")
+print(f"Mitigated result   {mitigated:.3f}")
+```
+
+As the ideal, desired result is `1.000`, the mitigated result performs much better than unmitigated.
+
+
+## Options
+
+In this section we will discuss the many different options for noise scaling and extrapolation that can be passed into PennyLane's `mitigate_with_zne` function.
+
+The following code block shows an example of using linear extrapolation with five different (noise) scale factors.
+
+```{code-cell} ipython3
+from mitiq.zne.inference import LinearFactory
+
+scale_factors = [1.0, 1.5, 2.0, 2.5, 3.0]
+noise_scale_method = fold_global
+mitigated = qml.transforms.mitigate_with_zne(
+    device_circuit,
+    scale_factors,
+    noise_scale_method,
+    LinearFactory.extrapolate, 
+)()
+
+print(f"Mitigated result {mitigated:.3f}")
+```
+
+To specify a different noise scaling method, we can pass a different function for the argument ``scale_noise``. This
+function should input a circuit and scale factor and return a circuit. The following code block shows an example of
+scaling noise by global folding (instead of local folding, the default behavior for
+{func}`zne.execute_with_zne`).
```

Comment:
```suggestion
function should input a circuit and scale factor and return a circuit. The following code block shows an example of
scaling noise by local folding instead of global folding.
```